contracts/api3-server-v1/Api3Market.sol

```diff
@@ -650,7 +743,7 @@ contract Api3Market is HashRegistry, ExtendedSelfMulticall, IApi3Market {
             );
             (beaconValues[0], beaconTimestamps[0]) = IApi3ServerV1(api3ServerV1)
                 .dataFeeds(deriveBeaconId(airnode, templateId));
-        } else {
+        } else if (dataFeedDetails.length != 0) {
```

Comment:
@peterjurco For you to be able to do [this](https://api3workspace.slack.com/archives/C03493F86Q7/p1709890163540239), this change is needed. You won't be able to do it with the current deployment.